File: qlty-cli/tests/lang/php/basic.in/cyclomatic/iterative_methods_with_filter_and_include.php
<?php
  function foo() {
    $animals = ["dog", "cat", "bear", "tiger"];

    array_filter($animals, function($animal) {
      return strlen($animal) > 3;
    });

    foreach ($animals as $animal) {
      echo $animal . "\n";
    }

    if (in_array("cat", $animals)) {
      echo "Found a cat!\n";
    }
  }
?>

File: qlty-cli/tests/lang/csharp/basic.in/cyclomatic/IterativeMethodsWithFilterAndInclude.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace Cyclomatic
{
  class IterativeMethodsWithFilterAndInclude {
    public static void Main(string[] args) {
      List<string> animals = new List<string> { "dog", "cat", "bear", "tiger" };

      animals.Where(animal => animal.Length > 3)
             .ToList()
             .ForEach(animal => Console.WriteLine(animal));

      if (animals.Contains("cat")) {
        Console.WriteLine("Found a cat!");
      }
    }
  }
}